greeter/README.md

```diff
@@ -418,23 +423,23 @@ this.configure('greeters.ChineseGreeter').to({nameFirst: false});
 
 ### Step 5: Try it Out!
 
-Restart your application, try endpoint `GET /greeting/{name}`, with `Accept-Language` set to `zh`.
+Restart your application, try endpoint `GET /greeting/{name}`, with `language` set to `zh`.
 
-You will see message `'name, hello'` printed instead of `'hello, name'`.
+You will see message `'hello, name'` printed instead of `'name, hello'`.
 
 Related documentation:
 [Configuration by convention](https://loopback.io/doc/en/lb4/Context.html#configuration-by-convention)
 
 ## Part 3: Enable Caching
 
-To reduce the cost of calculation, we'd like to use a caching server for our greeting application. We will show how to bind a caching server, interceptor and observer for your application as an example of contributing microservice using LB4.
+To reduce the cost of calculation, we'd like to use a caching service for our greeting application. We will show how to bind a caching service, interceptor and observer for your application as an example of contributing a microservice using LB4.
```

Comment:
hmmm. Our app greeting-app module can be considered a microservice. Our we saying it is not a microservice, but the caching service is?

I think we should mention the term microservice at the top in the intro section, and clarify that the greeter-app is the microservice.  @jannyHou @agnes512 what do you think? Because technically, caching service cannot be deployed as a microservice (application) in its present state.

Reply:
good point, modified